src/client/components/material-icon/index.js

```diff
@@ -19,6 +19,9 @@ class Icon extends PureComponent {
         <i
           className={cxName}
           dangerouslySetInnerHTML={{ __html: `&#x${iconCode};` }}
+          {...(onClick ? {onClick} : {})}
+          role="button"
```

Comment:
I'm not a fan of making the icon act like a button. A better approach would be to wrap the whole summary item in a `<button>`.